apis/apps/v1/cluster_types.go

```diff
@@ -426,6 +426,11 @@ type ClusterComponentSpec struct {
 	// +optional
 	ParallelPodManagementConcurrency *intstr.IntOrString `json:"parallelPodManagementConcurrency,omitempty"`
 
+	// Provides fine-grained control over the RollingUpdate process when a new ServiceVersion specified.
+	//
+	// +optional
+	RollingUpdate *RollingUpdate `json:"rollingUpdate,omitempty"`
```

Comment:
Maybe `RollingUpdateStrategy`. And the comment should be more accurate (i.e. it takes effect not only when ServiceVersion is changed).